README.md

```diff
@@ -27,3 +27,79 @@ You will need to configure the following environment variables and make them ava
 Also, for publishing to GitHub, you will need to provide a GitHub API key.
 
 For that, in travis, you can use the `env`. If your project is public, make sure to use [secrets](https://docs.travis-ci.com/user/encryption-keys/).
+
+## Tasks
+
+### Tests packages
+
+There are thee rules available for testing:
+
+* `test`: regular execution of the tests
+* `test-race` : execute the tests with the race detector enabled
+* `test-coverage`: execute the tests and get coverage. This rule generates a `coverage.txt` file that can be uploaded to codecov.io using the `codecov` rule.
+
+### Building packages
+
+The rule `packages` creates the distribution packages, containing the command
+line utilities defined by the `COMMANDS` compiled for the architectures and
+OS defined by `PKG_OS` and `PKG_ARCH`.
+
+The package filename is follows the following pattern: `<project>_<version>_<os>_<arch>.tar.gz`
+
+### Docker build and push
+
+The `docker-build` rule builds the dockerfiles defined by `DOCKERFILES`. Several
+dockerfiles may be define in a project eg.:
+
+If we have an application with two different docker images, we can achieve this
+by creating two different docker files called `Dockerfile.server` and
+`Dockerfile.client`.
+
+```
+DOCKERFILES = Dockerfile.server:$(PROJECT)-server Dockerfile.client:$(PROJECT)-client
+```
+
+The `DOCKERFILES` variable should be defined as list of file/name pairs, for example `Dockerfile:my-image`
```

Comment:
file/name -> dockerfile-path:image-name
also add a `.` at the end of the sentence.